### PR TITLE
OCPBUGS-6714: Initialize egress node monitoring struct with previous reachability status

### PR DESCRIPTION
**Pull request by kyrtapz**
There are two issues this PR tries to address:

[1] When egress node goes offline and there is **only one** egress IP assigned to a netnamespace the flows will not be removed from the node hosting a matching pod: 
    - in auto mode, the flow will get cleared once the sdn-controller removes the egress IP from the hostsubnet
    - in manual mode, the pod traffic will still be sent to the egress node

[2] When one egress node goes offline and there is **multiple** egress IPs assigned to a netnamespace the following could happen: 
1. On the pod node SDN detects the egress node as offline, sets `egressNode.offline` to true, and modifies the OVS flows:
```
I0210 13:55:25.625292  148815 egressip.go:350] Node 10.0.130.175 may be offline... retrying
I0210 13:55:27.626969  148815 egressip.go:350] Node 10.0.130.175 may be offline... retrying
W0210 13:55:29.627456  148815 egressip.go:345] Node 10.0.130.175 is offline
I0210 13:55:29.627493  148815 egressip.go:678] VNID 0 cannot use egress IP 10.0.128.101 on offline node 10.0.130.175
```
2. sdn-controller removes the affected egress IP from the hostsubnet:
```
I0210 13:55:37.622737  148815 egressip.go:238] Watch MODIFIED event for HostSubnet "ip-10-0-130-175.ec2.internal"
I0210 13:55:37.622791  148815 subnets.go:47] Watch MODIFIED event for HostSubnet "ip-10-0-130-175.ec2.internal"
I0210 13:55:37.628057  148815 egressip.go:643] Removing egress IP 10.0.128.101 from node 10.0.130.175
I0210 13:55:37.628079  148815 egressip.go:293] Unmonitoring node 10.0.130.175
I0210 13:55:37.628084  148815 egressip.go:676] VNID 0 cannot use unassigned egress IP 10.0.128.101
```
  5. Egress node comes online.
  6. sdn-controller add the egress IP back to the node.
  8. On the pod node SDN calls `ClaimEgressIP` but does not adapt the flows because `egressNode.offline` is still set to true:
```
I0210 13:55:43.643621  148815 egressip.go:238] Watch MODIFIED event for HostSubnet "ip-10-0-130-175.ec2.internal"
I0210 13:55:43.643631  148815 subnets.go:47] Watch MODIFIED event for HostSubnet "ip-10-0-130-175.ec2.internal"
I0210 13:55:43.649829  148815 egressip.go:639] Assigning egress IP 10.0.128.101 to node 10.0.130.175
I0210 13:55:43.649848  148815 egressip.go:271] Monitoring node 10.0.130.175
I0210 13:55:43.649852  148815 egressip.go:678] VNID 0 cannot use egress IP 10.0.128.101 on offline node 10.0.130.175
```

To address the first issue always check whether the egress node is offline or not before configuring the OVS flows, regardless of the number of egress IPs assigned to the netnamespace. 
To address the second issue use `egressNode.offline` to initialize the egress node monitoring struct so SDN can react to reachability change properly.

**Reproduction steps (the issue doesn't always occur):**
1. Create a namespace with a pod.
2. Patch two hostsubnets with `egressCIDRs` (don't use the pod node).
3. Assign two egress IPs to the previously created namespace.
4. Reboot the first egress node and wait for the egress IP to get assigned back to it.
5. Reboot the second egress node and wait for the egress IP to get assigned back to it.
6. Check external connectivity from a pod created in step 1.

Signed-off-by: Patryk Diak <pdiak@redhat.com>

### CODE REVIEW

**Comment by danwinship** [comment]
These both seem right
/lgtm

File: pkg/network/master/egressip.go [hunk 1]
```diff
@@ -197,9 +197,7 @@ func (eim *egressIPManager) maybeDoUpdateEgressCIDRs() (bool, error) {
 }
 
 const (
-	pollInterval   = 5 * time.Second
```
Comment (flavio-fernandes) [inline pkg/network/master/egressip.go:200]: @kyrtapz qq on this particular commit: the reason for moving it is to remove the duplication that we observe in this file and in pkg/network/node/egressip.go, correct?

**Pull request merged**